library_of_babel.py

```diff
@@ -16,6 +16,32 @@
 #letters per page: 3239
 #titles have 25 char
 
+help_text = '''
+checkout <addr> -- Checks out a page of a book. Also displays the page's title.
+
+search <text> -- Does 3 searches for the text you input:
+>Page contains: Finds a page which contains the text.
+>Page only contains: Finds a page which only contains that text and nothing else.
+>Title match: Finds a title which is exactly this string. For a title match, it will only match the first 25 characters. Addresses returned for title matches will need to have a page number added to the tail end, since they lack this.
+
+fsearch <file> -- Does exactly the search does, but with text in the file
+
+help -- Prints this message        '''
+
+
+def text_prep(text):
+    digs = 'abcdefghijklmnopqrstuvwxyz, .'
```

Comment:
Please use a set instead of a string for digs. Using a set will give you O(1) lookup.